clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-# Changelog
+# HLASM Language Support Changelog
```

Comment:
This is also maintained by the CI, everything will by updated before the file package is generated.